### PR TITLE
added linter integration into CI

### PR DESCRIPTION
**Pull request by imskr**
I have fixed issue #111

### CODE REVIEW

**Comment by imskr** [comment]
I have added the eslint to travis CI and there are some errors found in BookReader.js file and build is failing

File: .eslintrc.js [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
+    },
     "parserOptions": {
-        "ecmaVersion": 5
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "rules": {
     }
-};
+};
```
Comment (mekarpeles) [inline .eslintrc.js]: let's add newline here at the end of file

File: .eslintrc.js [hunk 2]
```diff
@@ -1,10 +1,18 @@
 module.exports = {
     "env": {
         "browser": true,
-        "jquery": true
```
Comment (mekarpeles) [inline .eslintrc.js]: Why did we remove jquery? This appears to perhaps be the cause of the issue:
https://travis-ci.com/internetarchive/bookreader/builds/117072482

File: .eslintrc.js (hunk 1 above)
Comment (mekarpeles) [inline .eslintrc.js]: newline at end of the file please

**Comment by mekarpeles** [comment]
Thanks for your contributions!

Build currently fails w/ missing reference to `jquery`:

```
/home/travis/build/internetarchive/bookreader/BookReader/BookReader.js
    33:15  error  'jQuery' is not defined  no-undef
   310:19  error  'jQuery' is not defined  no-undef
  3061:5   error  'jQuery' is not defined  no-undef
  3069:5   error  'jQuery' is not defined  no-undef
  4777:1   error  'jQuery' is not defined  no-undef
  4778:12  error  'jQuery' is not defined  no-undef
  4783:4   error  'jQuery' is not defined  no-undef
```

**Comment by imskr** [comment]
@mekarpeles I've added the jquery line and also added newline at the end. Now it works!

**Comment by jbuckner** [comment]
@imskr Would you be interested in joining the Internet Archive's BookReader slack channel?

**Comment by imskr** [comment]
Yeah I would love to!  Please share the link! @jbuckner

**Pull request merged**